packages/sample-app/__tests__/__snapshots__/webpack-config.test.js.md

```diff
@@ -9,33 +9,16 @@ Generated by [AVA](https://ava.li).
 > Snapshot 1
 
     {
-      devServer: {
```

Comment:
Is that correct? ... 🤔

Reply:
It changed when I only updated `ava` from 0.22.0 to 0.25.0 before doing any other changes. I couldn't find any [release notes](https://github.com/avajs/ava/releases) related to it.

I'll have look on how to fix it.

Reply:
Fixed by setting `NODE_ENV` to `development`. This was broken because [since 0.23.0 `ava` sets `NODE_ENV` to `test`](https://github.com/avajs/ava/releases/tag/v0.23.0).